src/components/menu-top/MenuTop.vue

```diff
@@ -14,6 +14,14 @@
 
           <hr class="navbar-divider" />
 
+          <b-navbar-item
+            data-test="menu-top--save-show"
+            :href="fileURL"
```

Comment:
Note that a downside is that whenever `show` is modified, it has to re-compute `fileURL`... I think this is fine for now, but it would be good to see if we can compute `fileURL` only when the user clicks "Export Show".

Reply:
With calchart 3 we have a `dummy` file that gets saved every 1 minute in case there's a crash -- that way there's a recoverable file.  Maybe this isn't so bad to have to start off with.